docs/zh_CN/admin-api/README.md

```diff
@@ -0,0 +1,131 @@
+# Management REST API
+
+## Authentication
+- AWS signatureV4
+- We use "minio" as region. Here region is set only for signature calculation.
```

Comment:
@dingjs  will this doc be translated as well?

Reply:
All the files in zh_CN will be translated.If docs.minio.com has a zh_CN language option,maybe more chinese developers will participate and speed up the translation.